README.md

```diff
@@ -6,7 +6,7 @@
 
 <p align="center"><code>chartmogul-php</code> provides convenient PHP bindings for <a href="https://dev.chartmogul.com">ChartMogul's API</a>.</p>
 <p align="center">
-  <a href="https://packagist.org/packages/chartmogul/chartmogul-php"><img src="https://badge.fury.io/ph/chartmogul/chartmogul-php.svg" alt="PHP Package" /></a>
+  <a href="https://packagist.org/packages/chartmogul/chartmogul-php"><img src="https://badge.fury.io/ph/chartmogul%2Fchartmogul-php.svg" alt="PHP Package" /></a>
```

Comment:
Something to do with how fury.io have set this up, in their system the package name is actually `chartmogul/chartmogul-php` so the `/` in there doesn't indicate a directory.